shell/platform/fuchsia/flutter/vulkan_surface.cc

```diff
@@ -70,6 +61,7 @@ bool CreateVulkanImage(vulkan::VulkanProvider& vulkan_provider,
       .arrayLayers = 1,
       .samples = VK_SAMPLE_COUNT_1_BIT,
       .tiling = VK_IMAGE_TILING_OPTIMAL,
+      // We should only keep usages that are actually required by Skia.
       .usage = VK_IMAGE_USAGE_COLOR_ATTACHMENT_BIT |
                VK_IMAGE_USAGE_TRANSFER_DST_BIT |
                VK_IMAGE_USAGE_TRANSFER_SRC_BIT | VK_IMAGE_USAGE_SAMPLED_BIT,
```

Comment:
nit: start comment with FIXME or TODO so it's clear that it should be improved by follow up change
nit: can you also move these to a constant at the top of the file so it's easier to see that sysmem and vulkan usages should both be adjusted?